tools/winmd2objc/include/WinRTSymbolTable.h

```diff
@@ -34,7 +34,7 @@ class SymbolNode;
 //  intentional - if we ever hit the templated version, it's an error.
 template <class T>
 bool CompareStringLess(T left, T right) {
-    NT_ASSERT(FALSE);
+    assert(FALSE);
```

Comment:
is there any substantive difference b/w assert and NS_ASSERT? Should we be using something with telemetry?